src/amo/components/SearchPage.js

```diff
@@ -45,9 +46,12 @@ export default class SearchPage extends React.Component {
 
     return (
       <div className="SearchPage">
+        {hasSearchParams ? (
```

Comment:
> Adding bits that only appear once there are results to filter seems like bad UI/UX to me.
> 
> I should change this to check for results as well though; no sense in showing it when there isn't anything to sort.

I feel I was suggesting the same thing as the first sentence, but then the second one says it should only appear when there are results 😕

It does seem a bit pointless to include it when there are no results to filter but just always having it there makes sense to me too. Maybe you want to set it to "Top rated" and then enter your search term.

In the end it doesn't matter much but I would lean towards the consistency of it always being there.